docs/src/_includes/layouts/base.html

```diff
@@ -121,6 +121,7 @@
     <script type="module" src="{{ '/assets/js/search.js' | url }}"></script>
     <link rel="stylesheet" type="text/css" href="{{ '/assets/css/styles.css' | url }}">
     <link rel="stylesheet" href="{{ '/assets/css/print.css' | url }}" media="print">
+    <script src="https://unpkg.com/anchor-js@4.3.1/anchor.min.js"></script>
```

Comment:
This was an actual dependency in package.json on the [old documentation site](https://github.dev/eslint/archive-website/blob/master/package.json#L33) so probably it should be here as well?

Reply:
Yea? We could do it either way? https://www.npmjs.com/package/anchor-js both are suggested as per their documentation?

Reply:
Personally I prefer to take advantage of the package management provided by package.json and specify it there, especially given that we haven't begun using unpkg anywhere.

Reply:
we have to change the imports as `module` if we want to use imports or add passthrough through eleventy (I haven't tried these yet) anyway the passthrough is just going to import add the files into the assets and we will be importing it from there. I don't think there is going to be much difference using the dep from npm too instead of this approach which seems very straight forward.

Reply:
I think this is fine for now. If we end up needing to upgrade this dependency at any point, then I'd say let's move it to an npm dependency.